sqlglot/expressions.py

```diff
@@ -2055,6 +2056,23 @@ class Credentials(Expression):
     }
 
 
+class DistributedByHash(Expression):
+    arg_types = {"expressions": True, "buckets": False, "sorted_by": False}
+
+    @property
+    def auto_bucket(self) -> bool:
+        return self.args.get("buckets") is None
+
+
+# https://docs.starrocks.io/docs/sql-reference/sql-statements/data-definition/CREATE_TABLE/#distribution_desc
+class DistributedByRandom(DistributedByHash):
+    pass
```

Comment:
Why is this inheriting from `DistributedByHash`?

Reply:
Their arg_types are the same, which is probably not good here, I added that in their respective classes instead.

Reply:
Yep, that by itself is not a good reason to inherit. You could use a mixin and inherit from that, but I'd just replicate the args because they're only two expressions.